test/old-start-configs/v1.0.0/test-end-to-end.sh

```diff
@@ -0,0 +1,371 @@
+#!/bin/bash
+
+# this is effectively the e2e from v1.0.0.  It is expected to run forever
```

Comment:
Probably highlights that we need better e2e test structure - a lot of these tests are probably too indepth.  I suppose we can reuse this for later revisions, although I'd want to reuse the certs as much as possible across versions.

Did you consider parameterizing the regular end to end to bypass the config directory, and just expect the tests to pass?  After all, we want new code to start with old config - is this copy of the e2e test more important than the new code?